packages/core/core-flows/src/cart/workflows/refresh-cart-items.ts

```diff
@@ -228,6 +229,8 @@ export const refreshCartItemsWorkflow = createWorkflow(
       },
     })
 
+    createHook("beforeRefreshingPaymentCollection", { input })
```

Comment:
Also to be added to the TSDocs (and pass it to in the WorkflowResponse, unless we don't want to show it to users, as the workflow reference requires the hook to be accessible as a type)